.cargo/config.toml

```diff
@@ -3,3 +3,6 @@ run_wasm = "run --release --package run_wasm --"
 # Other crates use the alias run-wasm, even though crate names should use `_`s not `-`s
 # Allow this to be used
 run-wasm = "run_wasm"
+
+[target.'cfg(target_family = "wasm")']
+rustflags = ["--cfg=web_sys_unstable_apis"]
```

Comment:
This was needed with the new version of egui for the demo, because it uses the unstable clipboard API.